apps/test/unit/sites/studio/pages/levelbuilder_edit_script.js

```diff
@@ -21,6 +21,10 @@ describe("the level builder page init script", () => {
       },
       beta: false,
       levelKeyList: [],
+      locales: [
+        ["English", "en-US"],
+        ["French", "fr-FR"]
```

Comment:
Does this mean that levelbuilders will never have to select English and French as supported locales because they are automatically included? If so, will that be true for French even when we make brand new scripts?

Reply:
This is just for a unit test.  In reality, the levelbuilder will need to choose each non-`"en-"` language in the levelbuilder UI.  (The `"en-"` options will not be offered.)